.github/workflows/unittests.yml

```diff
@@ -28,3 +28,11 @@ jobs:
         make check
     - name: Run unit tests
       run: make test
+    - name: Generate artifacts
+      run: ./contrib/scripts/release.sh
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      if: success()
```

Comment:
does it have to be `if: ${{ success() }}`?

Reply:
no, amazing syntactic sugar:

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.

https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions

Reply:
I also followed their documentation and found the non-omition syntax in the example https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions